test/sort-issues-test.js

```diff
@@ -20,17 +20,18 @@ describe('sort-issues', function() {
       assert.deepEqual(sorted, [i1,i2,i3]);
    });
 
-   it('should sort issues according to ASC priority, then appliedOn DESC', function() {
-      var i1 = issue('p1', 3);
+   it('should sort issues by createdOn ASC', function() {
+      var i1 = issue('p1', 1);
       var i2 = issue('p1', 2);
-      var i3 = issue('p1', 1);
+      var i3 = issue('p1', 3);
```

Comment:
Might as well put an i4 in there that has the same `createdOn` as another one.

Reply:
I cover that case (same p-label and createdOn) in the test above.

Reply:
Oh, and in the case of same p, previous, and createdOn, the sort behavior is undefined, so there's not much to test.